datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -269,6 +269,9 @@ func (f *ChainDataFetcher) makeChainEvent(blockNumber uint64) (blockchain.ChainE
 		return blockchain.ChainEvent{}, fmt.Errorf("GetBlockByNumber is failed. blockNumber: %v", blockNumber)
 	}
 	receipts := f.blockchain.GetReceiptsByBlockHash(block.Hash())
+	if receipts == nil {
+		return blockchain.ChainEvent{}, fmt.Errorf("GetReceiptsByBlockHash is failed. blockNumber: %v", blockNumber)
```

Comment:
```suggestion
		return blockchain.ChainEvent{}, fmt.Errorf("GetReceiptsByBlockHash is failed. blockNumber=%v", blockNumber)
```
Usually in go, `:` is used for concatination multiple errors. I think would be better to change it to `=` like we the way we make logs.

Reply:
Reflected. Thanks.